package/src/fileSystem.js

```diff
@@ -100,7 +99,7 @@ const read = absolutePath =>
 
 // @TODO save got 4 attributes and most of them are about directory/files...
 // there should be another way
-const save = (folderNamePath, file, fileData, flag) => {
+const save = (folderNamePath, fileData) => {
```

Comment:
what was the reason of removing a flag variable? we're not using it?
I feel that we need it

Reply:
i checked again and discovered i shouldnt have remove both flag and file